uploadNeuroDB/tarchiveLoader.pl

```diff
@@ -591,7 +591,7 @@ =head2 Methods
     #### link the tarchive and mri_upload table  with session ##
     ############################################################
     my ($sessionRef, $errMsg) = NeuroDB::MRI::getSessionInformation(
-        $subjectIDsref, $tarchiveInfo{'DateAcquired'}, $dbh, $db
+        $subjectIDsref, $tarchiveInfo{'DateAcquired'}, \$dbh, $db
```

Comment:
I happen to find this one too when trying to resolve the bug and since it is related to the same function I included it in the same PR.